site/content/docs/scans/firingrange.md

```diff
@@ -31,8 +31,17 @@ Changes which find any of the missed vulnerabilities are eligible for a bounty v
 
 {{< /scan-table >}}
 
+#### Configuration
+
 | Config | Details |
 | --- | --- |
 | Frequency | Daily |
 | Scripts | https://github.com/zapbot/zap-mgmt-scripts/blob/master/scans/firingrange/ |
 | Action | https://github.com/zapbot/zap-mgmt-scripts/actions/workflows/zap-vs-firingrange.yml | 
+
+
+#### Non Default Settings
```

Comment:
For some reason in your screenshot this seems quite squished together, might be an explicit </br> or </p> :man_shrugging:

Reply:
Its CSS, I'll try to fix that, but its not my strong suit :/